pip/wheel.py

```diff
@@ -37,6 +37,12 @@
 from pip.utils.temp_dir import TempDirectory
 from pip.utils.ui import open_spinner
 
+try:
+    from concurrent.futures.process import ProcessPoolExecutor
```

Comment:
Don't try importing the unvendored version, it could break compatibility. :)

Reply:
Could you change this into a conditional on the Python version? That'll address my reservations.

Reply:
That's fine with me.